access/management/commands/pregenerate_exercises.py

```diff
@@ -42,14 +46,19 @@ def handle(self, *args, **options):
             # take only personalized exercises
             exercises = list(filter(lambda ex: "personalized" in ex and ex["personalized"], exercises))
             if not exercises:
-                raise CommandError("The course %s has no personalized exercises" % (course_key))
+                self.stderr.write("The course %s has no personalized exercises so no instances are generated." % (course_key))
```

Comment:
something like `self.stdout.write(self.style.INFO(...))` could be better. Also, no need to use stderr